main.go

```diff
@@ -462,3 +473,51 @@ func stopStalePipelines (client *gitlab.Client, vars []*gitlab.PipelineVariable)
 
 	}
 }
+
+// syncEnterpriseRepos detects whether a commit has been merged to
+// the Open Source edition of a repo, and then sends a GET request to
+// the Gallienii server, asking it to synchronize the repos
+func syncEnterpriseRepos(conf *config, gpr *github.PullRequestEvent) error {
+
+	pr := gpr.GetPullRequest()
+	if pr == nil {
+		return fmt.Errorf("syncEnterpriseRepos: Failed to get the pull request")
+	}
+
+	repo := gpr.GetRepo()
+	if repo == nil {
+		return fmt.Errorf("syncEnterpriseRepos: Failed to get the repository information")
+	}
+
+	// Enterprise repo sentinel
+	switch repo.GetName() {
+	case "deployments":
+	case "useradm":
+	default:
+		// Do not run sync, whenever there is no merge to an Enterprise repo
+		return nil
+	}
+
+	// If the action is "closed" and the "merged" key is "true", the pull request was merged.
+	// While webhooks are also triggered when a pull request is synchronized, Events API timelines
+	// don't include pull request events with the "synchronize" action.
+	if gpr.GetAction() == "closed" && pr.GetMerged() {
```

Comment:
We need to check that it has been merged into **master** and not to other release or feature branch.

Reply:
True!